### ISSUE TITLE
No artwork with radio streams

### ISSUE THREAD

**Issue by Meaparvitas111** [pssc/ha-addon-lms#94]
Hello,

during the last few days, I think since my update to lms 9.0, there is no cover art when listening to radio streams. Nor in a media player card nor on the display of a sqeezebox touch. The logo is sometimes shown, depeding on the station, but no artwork is shown. 
I read in the documentation that there is an issue with ingress, but I don't know what that means and what I could do about it.

Thanks and kind regards

Meaparvitas
![Unbenannt](https://github.com/user-attachments/assets/3a98dde4-3de0-42ee-afa6-83bba0b0c7bf)

**Comment by pssc** [comment]
So its not displaying on the touch itself?

**Comment by Meaparvitas111** [comment]
No, there is no cover art on the touch display either. It shows the same different pictures as the media player card, e. g. the station logo. Can you give me a hint? I cannot remember having changed anything in the settings before the issue occured. I just made the update.

**Comment by pssc** [comment]
So it may not be aunting you have done @Meaparvitas111 the station provider or radio plugin may have changed also, does LMS need a restart

**Comment by Meaparvitas111** [comment]
I have tried different stations and no one shows a cover art. I have also restartet Home Assistant with LMS an the addon several times.

**Comment by pssc** [comment]
@Meaparvitas111 give me you exact method to start a radio stream your having a problem with and I can see if i can replicate

**Comment by Meaparvitas111** [comment]
Thank you very much, I can choose a station via radio browser in home assistant media or via squeezebox, e. g. tunein radio or favorites. It is always the same issue.

**Comment by pssc** [comment]
So if say you go pick 80s Mixed does that work for you

**Comment by Meaparvitas111** [comment]
How do you mean? Can you please describe what I should try?Am 20.12.24, 14:14 schrieb "Phill (pssc)" ***@***.***>:

So if say you go pick 80s Mixed does that work for you

**Comment by Meaparvitas111** [comment]
I have found the station you mentioned.
The cover art is shown. I found out that it works if I choose a station via "Radio now playing". But if I search for a station using the other options "local radio", "music" a. s. o., only the station logo is shown. 
![IMG_20241221_105646516](https://github.com/user-attachments/assets/c3b01790-b185-4c1f-85e4-b778c3772646)
![IMG_20241221_105354115](https://github.com/user-attachments/assets/4ef3fd09-f5c9-41fa-8c0a-50f4c052dd4a)
![IMG_20241221_105510418](https://github.com/user-attachments/assets/6d64aae4-e427-40d3-ac74-efa9d9a9b6ce)


Can you tell me how I can fix that? I have installed "radio now playing" just recently when I was trying to solve the problem. Before that I always yoused tunein which is not showing cover art anymore.
Thanks and kind regards.

**Comment by Meaparvitas111** [comment]
I found out, that the sqeezebox was integrated twice, once over music assistant. When I select the device that is connected to music assistant no artwork is shown. When I select the device that is connected to LMS there is artwork, but not always an often not at once. Sometimes the artwork is shown for half a second when I change station and then the station logo is shown. After a while the artwork reapears in many cases. But I still do not understand the reason for this behaviour.
